validation/nsa/README

```diff
@@ -0,0 +1,37 @@
+# Monokle custom validator
+
+Welcome to Monokle custom validation plugin.
+
+Developing a plugin has is build with developer experience in mind and has a short feedback loop. Read more below.
```

Comment:
Should it be "Developing a plugin is built with"?

Reply:
Thanks, that was actually a bug in the skaffolding tool. I've updated it there as well!